### DISCUSSION TITLE
High CPU activity

### DISCUSSION THREAD

**Discussion by iesfausti** [openbroadcaster/obplayer#47] [Q&A] [answered]
Hi again,
Not really  a code-bug but a problem for a low-end htpc, the gstreamer-libs stuff make the system go high (CPU 60% usage in a static image splash) so may be a temporal, until the improve of their libraries, solution bypass for simple pictures, not video of course,its to stop the visual in aprox.271 line in control.py
`if req['media_class']==u'visual': #+only visual! not videos, music...`
`request_pipe.start() #+ Paint-it`
`time.sleep(1) #+ wait...)` 
`request_pipe.stop() #+ stop refresh)`
 `else:`
`request_pipe.start() #original, only one code`

By the way is the the 'testing branch available', because of the another annoying bug 
https://github.com/openbroadcaster/obplayer/issues/16

Thank-you!

**Comment by transistorfet** [comment, marked as answer]
This is a known issue with image playback.  We are actually converting the image to a video on the fly and outputting that video in order to allow for fade-in/fade-out effects on the image, and to allow the emergency alert overlay to be displayed on top of the image.

In the dashboard, on the audio/visualization tab, there is a section for Image Output Settings.  The images will be scaled to those width and height dimension settings, and the video will be generated with that framerate setting.  You can try reducing those or disabling image transitions (the fade-in/fade-out effect) in order to reduce the load on the CPU.  If you make the framerate setting very low, it will speed things up but you will also notice the emergency overlay will scroll less smoothly.  This is because the emergency overlay is drawing directly on top of each video frame rather than trying to mix the two video sources which is even harder on the CPU.
